.github/workflows/reuse.yml

```diff
@@ -0,0 +1,20 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+# SPDX-FileCopyrightText: 2022 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+name: REUSE Compliance Check
+
+on: [push, pull_request]
```

Comment:
```suggestion
on: pull_request
```

Reply:
Would it still get executed when you push further commits to a branch with an open PR?

Reply:
yes, push is for pushing to master, etc which is not allowed anyway or no one looks at the CI result.

Reply:
👍 fixed in the workflow repo via the SPDX PR 👍